audit-service/src/test/java/uk/gov/gchq/palisade/service/audit/request/AuditMessageTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2019 Crown Copyright
```

Comment:
```suggestion
 * Copyright 2020 Crown Copyright
```
Also needs to be checked for the other AuditMessageTest files